faq.rst

```diff
@@ -0,0 +1,22 @@
+Frequently Asked Questions
+==========================
+
+.. note::
+
+   For questions related to biling, please head to the :doc:`billing` section.
+
+What is accounted for storage quota?
+------------------------------------
+
+Your storage quota (as show by ``symfony project:info``) grants a maximum
+storage capacity per environment. For example, the default project storage quota
+of 5GB grants 5GB for each environment created.
+
+This quota is then shared by your :doc:`services </services/intro>` and
+:doc:`applications </config>`. Each one defines its own quota using the ``disk``
+configuration key. The sum of these quotas can not be bigger than your global
+project quota, in such case, an error message will be shown during ``symfony
+deploy``.
+
+The space used to store your source code, your application container images, and
+the temporary files (``/tmp``) does not account for the storage quota.
```

Comment:
so not

Reply:
I don't understand your review 🤔

Reply:
do not account

Reply:
Hum, "The space used [...] does not account". no?

Reply:
nvm, you're right